build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -66,7 +61,6 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
@livarcocc What do you mean clean up the Compile and EmbeddedResource items? You mean combine them (into one Compile, and one EmbeddedResource)? Or something else?